main.py

```diff
@@ -286,6 +319,44 @@ def validateSettings(staticLetters) -> bool:
     return True
 
 
+def normalized(sett: dict) -> list:
+    total = sum(sett[i] for i in sett)
+    return [(sett[i]/total, i) for i in sett]
+
+
+def orderLetters(configs: tuple = BIGRAMS_CONFIGS) -> str:
+
+    # Generate monograph sets for each language
+    monoSets = []
+    for config in configs:
+        if config.weight <= 0:
+            continue
+        with open(config.path, 'r') as biFile:
+            bis = {line[:2]: float(line[3:]) for line in biFile}
+        notNorm = {}
```

Comment:
Using `x = {}` is a perfectly reasonable practice, but for the sake of consistency, please replace all `{}` with `dict()`.

Reply:
(This includes `bis = {...}`)

Reply:
dictionary comprehension doesn't work with the `dict(...)` syntax

Reply:
Done in 444e09cfcf29a2dc6c1ad61332d4f2e98f975ae6
Though I have left the `{... for ... in ...}` syntax